simulator/risc_v/t/unit_test.cpp

```diff
@@ -353,6 +354,9 @@ TEST_RV32_RR_OP( 3, minu, 0x22222222, 0xbbbbbbbb, 0x22222222)
 TEST_RV32_RR_OP( 1, packu, 0x1111ffff, 0xffff2222, 0x11113333)
 TEST_RV64_RR_OP( 1, packu, 0x11111111ffffffff, 0xffffffff22222222, 0x1111111133333333)
 
+TEST_RV64_RR_OP( 1, add_uw, 0x1111111155555555, 0x5555555522222222, 0x1111111133333333)
+TEST_RV64_RR_OP( 2, add_uw, 0x12344321b5a69788, 0xabababab82736455, 0x1234432133333333)
```

Comment:
This test case is just obfuscated version of the test case above. Not sure if we need it.